toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -814,6 +814,12 @@ func customizeVerityImageHelper(buildDir string, baseConfigPath string, config *
 		return fmt.Errorf("failed to stat file (%s):\n%w", grubCfgFullPath, err)
 	}
 
+	// Refresh disk partitions so that the hash partition's UUID is correct
```

Comment:
It would be good this closer to the `veritysetup` call so that the relationship between the two is clearer.

Reply:
I located it right before the `updateGrubConfigForVerity` that uses the updated diskPartitions value, but happy to move it